native/bin/main/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json

```diff
@@ -0,0 +1,229 @@
+[
```

Comment:
We don't need the native image config files right?

Reply:
It was a mistake during merge. Fixed in [ccc55c2](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/734/commits/ccc55c2998309ea1dd0374132ac8b1b08c6c2d2d)